host/sgx/sgxquoteprovider.h

```diff
@@ -25,7 +25,13 @@ typedef struct _oe_sgx_quote_provider
         free_sgx_quote_verification_collateral;
 } oe_sgx_quote_provider_t;
 
+bool oe_set_quote_provider_logger(
+    oe_sgx_quote_provider_t* provider,
+    sgx_ql_logging_function_t logger);
```

Comment:
This header file has no other oe_sgx_* functions in it.  I don't follow why we need this in the OE SDK per se.  If you know it's SGX specific, can you use a lower level API?   If you need it in the OE SDK, can it be made into a TEE-agnostic API?

Reply:
The reality is that this file is SGX specific, we may need a refactor for this file to move non SGX specific APIs out of this file, (the file name is actually sgx prefixed and it's located in sgx specific directory). For now, I renamed this API to sgx specific, thinking to remove this API, but need to keep the existing tools compatibility